exercises/ocr-numbers/description.md

```diff
@@ -38,10 +38,10 @@ Update your program to recognize multi-character binary strings, replacing garbl
 Update your program to recognize all numbers 0 through 9, both individually and as part of a larger string.
 
 ```text
```

Comment:
This needs to be ignored.

Reply:
The text block?

Reply:
yes, it contains relevant trailing whitespace.

Reply:
`<!-- prettier-ignore -->`

If it's multiple blocks:

`<!-- prettier-ignore-start-->`
`<!-- prettier-ignore-end -->`